### ISSUE TITLE
Request/Bug

### ISSUE THREAD

**Issue by mikhoul** [nulled666/nppqcp#35]
Why in JS the color highlight don't work ? 

If it's intended it would be very useful to have an option to enable it in JS.

I do write lot of Userscript and I have lot of CSS in my JS to intergrate so the highlight would be very useful to have.

Best Regards !   :octocat:

**Comment by CollinChaffin** [comment]
I agree this is a bug the option is listed and able to be enabled and just seems to not do anything.

**Comment by CollinChaffin** [comment]
FYI, last coding/commit activity at all from the owner of this repo was July of 2016.  He's shown up to just blatantly close issues for some reason he is refusing to fix, so I would recommend just forking this yourself as it does not appear to have any active development.
